apps/BasicExample/ios/Podfile

```diff
@@ -2,11 +2,9 @@
 require Pod::Executable.execute_command('node', ['-p',
   'require.resolve(
     "react-native/scripts/react_native_pods.rb",
-    {paths: [process.argv[1]]},
+    {paths: ["../../../node_modules"]},
   )', __dir__]).strip
 
-require_relative '../../scripts/clangd-add-xcode-step.rb'
```

Comment:
I guess, this will ultimately change location, no?

Reply:
Yes, it will 😅